src/CMakeLists.txt

```diff
@@ -198,6 +198,7 @@ set(autotype_SOURCES
     core/Tools.cpp
     autotype/AutoType.cpp
     autotype/AutoTypeAction.cpp
+    autotype/AutoTypePickCharsDialog.cpp
```

Comment:
Because you are now using config() and FilePath in the autotype built library, you need to add ```keepassx_core``` to line 224 of this file ```target_link_libraries(autotype Qt5::Core Qt5::Widgets keepassx_core)```

Reply:
I don't know, `autotype/AutoTypeSelectDialog.cpp` (included 2 line below this comment) also uses `config.h` and `FilePath.h` and gave no problem before

Reply:
It doesn't make much sense to me either but i did a clean build with the original commits from this pr and it still failed.

Reply:
I can confirm that this builds and appears to work correctly for me. I'm guessing the difference with `autotype/AutoTypeSelectDialog.cpp` is that that dialog is used during initialisation before the auto-type has begun, while mine is shown while the auto-type is being performed so is probably being called from a different place.